pkg/proxy/service.go

```diff
@@ -109,6 +110,9 @@ func (info *BaseServiceInfo) ExternalIPStrings() []string {
 // LoadBalancerIPStrings is part of ServicePort interface.
 func (info *BaseServiceInfo) LoadBalancerIPStrings() []string {
 	var ips []string
+	if info.disableExternalShortRouting {
+		return ips
```

Comment:
currently AWS ELB works correctly, because it do have only `ing.cname` field not `ing.ip`. The problem do exist in all cloudproviders which uses `ing.IP` to expose ip address